packages/dd-trace/src/pkg.js

```diff
@@ -22,7 +22,11 @@ function findPkg () {
 
 function findUp (name, cwd) {
   let directory = path.resolve(cwd)
-  const { root } = path.parse(directory)
+  const res = path.parse(directory)
+
+  if (!res) return
```

Comment:
There should probably be a similar check in `findPkg` since otherwise it relies on the filesystem operation to fail to return the default value. Some of this file could maybe be cleaned up to avoid multiple checks as well.